scriptmodules/ports/steamlink.sh

```diff
@@ -33,4 +33,12 @@ function remove_steamlink() {
 
 function configure_steamlink() {
     addPort "$md_id" "steamlink" "Steam Link" "/usr/bin/steamlink"
+
+    # RetroPie sets overscan by default, which requires an override
+    if egrep '^overscan_scale=1' /boot/config.txt >/dev/null 2>&1; then
+        sudo -u $user bash -c "\
+            mkdir -p "~/.local/share/SteamLink/"
+            touch "~/.local/share/SteamLink/.ignore_overscan"
+        "
```

Comment:
mkUserDir and touch/chown to match rest of code please.

Reply:
actually mkUserDir isn't ideal, but we don't use sudo like this in other code, so it would be best to stick with the existing style (mkUserDir .local / .share etc - it's longwinded but perhaps we could improve that function later).

Reply:
probably best to make this folder by default anyway - and then just set the .ignore_overscan as part of this logic.

Reply:
I'm not sure if moveConfigDir is appropriate in this case, as `~/.local/share/SteamLink` is where the actual binaries are downloaded and updated too. There is also a `~/.local/share/Valve Software/Streaming Client/` dir which contains autogenerated configuration based on the active session, but is also the location to put an optional user configuration (streaming_args.txt). Maybe that file can be linked via moveConfigFile at least?

Reply:
Ah I see. Hmm. I'll leave it up to you :-)

Reply:
All your comments should be addressed.

I've only added streaming_args.txt via moveConfigFile, and intentionally left it blank, as adding unrecognized parameters - including commented lines - will break execution of the streaming_client binary.

Users will be able to dig into the supported arguments via a Google search, or perhaps someone will create a wiki entry on our site.